src/webview/js/spoilers.js

```diff
@@ -0,0 +1,120 @@
+/* @flow strict-local */
+/* eslint-disable no-useless-return */
```

Comment:
One more nit which I'd spotted but forgot to mention: this line doesn't seem needed.

It's there in js.js because of the click handler's structure: db4631fa55124248adeaed4447c49501b450e84f.
But this file doesn't have any code that looks like that, and it seems to pass lint without this line.

(I'm not sure this lint rule really does much good in general. But if we were to disable it generally, we still wouldn't need this line in this particular file.)